cmd/meroxa/turbine/ruby/build.go

```diff
@@ -0,0 +1,17 @@
+package turbinerb
+
+import (
+	"context"
+)
+
+// Build has nothing to do for turbine-rb.
+func (t *turbineRbCLI) Build(ctx context.Context, appName string, platform bool) (string, error) {
+	return "", nil
+}
```

Comment:
Isn't this important for building a function? So it is something that we should be doing here